bzt/requests_model.py

```diff
@@ -245,6 +247,20 @@ def _parse_request(self, req):
 
     def extract_requests(self, require_url=True):
         requests = self.scenario.get("requests", [])
+
```

Comment:
This function should probably be recursive to correctly unwrap nested transactions. Right now it only unwraps the first level.